src/views/3-Modeling/Modeling.vue

```diff
@@ -4597,7 +4597,7 @@ figure.sticky.charts {
   height: 90vh;
   width: auto;
   @media screen and (min-height: 770px) {
-        grid-template-rows: 40% 60%;
+        grid-template-rows: 35% 65%;
         grid-template-columns: 1.5fr 3fr 2%;
```

Comment:
oh didn't realize these didn't have to be whole numbers. I've really be overcomplicating my fraction math in snow-to-flow...

Reply:
I don't really get the notation, but it seems like whatever numbers you use, their sum becomes the denominator. So you can make the fractional numbers whatever you want? At least that's what I took from this description I found:
>The fr unit allows you to set the size of a track as a fraction of the free space of the grid container. For example, this will set each item to **one third** the width of the grid container:
> `.container {
  grid-template-columns: 1fr 1fr 1fr;
}`
So there, the sum is 3, making each of those fractionally represent 1/3

Reply:
By that logic, my way of including values other than whole numbers might actually make it more confusing? Not sure